### PR TITLE
Quenching some bugs

### PR DESCRIPTION
**Pull request by gklps**

### CODE REVIEW

File: src/com/rubix/TokenTransfer/TransferPledge/Pledger.java [hunk 1]
```diff
@@ -15,242 +15,253 @@
...
-                    executeIPFSCommands(" ipfs p2p close -t /p2p/" + senderPID);
-                } catch (IOException e) {
-                    executeIPFSCommands(" ipfs p2p close -t /p2p/" + senderPID);
-                    PledgerLogger.error("IOException Occurred", e);
-                }
-
-            }
-        }
-    }
+	public static Logger PledgerLogger = Logger.getLogger(Pledger.class);
```
Comment (vyshnavsdeepak) [inline src/com/rubix/TokenTransfer/TransferPledge/Pledger.java:18]: Looks like this file has unnecessary changes - just white space addition

Comment (vyshnavsdeepak) [reply src/com/rubix/TokenTransfer/TransferPledge/Pledger.java:18]: Looks like 4spaces is the standard here, then I guess this is okay.

File: src/com/rubix/TokenTransfer/TokenReceiver.java [hunk 2]
```diff
@@ -95,18 +95,17 @@ public static String receive() {
...
 			JSONObject final_parttokenchains = new JSONObject();
-			
 
 			String receiverPeerID = getPeerID(DATA_PATH + "DID.json");
 
 			String receiverDidIpfsHash = getValues(DATA_PATH + "DataTable.json", "didHash", "peerid", receiverPeerID);
 
 			JSONObject temp = new JSONObject();
-			temp.put("pvtShare",DATA_PATH + receiverDidIpfsHash + "/PrivateShare.png");
+			temp.put("pvtShare", DATA_PATH + receiverDidIpfsHash + "/PrivateShare.png");
```
Comment (KiranHRubix) [inline src/com/rubix/TokenTransfer/TokenReceiver.java:108]: Is this temp object required?

**Review by KiranHRubix — APPROVED** [review]
